### PR TITLE
fix conda packages registry & add tests

### PR DESCRIPTION
**Pull request by mrMakaronka**

### CODE REVIEW

File: lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java [hunk 1]
```diff
@@ -43,16 +47,17 @@ public static boolean isInstalled(String condaYaml) {
...
-            for (var dep: deps) {
+            for (var dep : deps) {
 
                 if (!(dep instanceof String) && !(dep instanceof Map)) {
                     return false;
                 }
 
                 if (dep instanceof String) {
-                    var res = ((String) dep).split("=");
+                    var res = ((String) dep).split(VERSION_REGEX, SPLIT_LIMIT);
```
Comment (imakunin) [inline lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java:60]: nit: 
```java
if (dep instanceof String s) {
  var res = s.split(...);
  ...
}
```

File: lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java [hunk 2]
```diff
@@ -144,21 +149,24 @@ public static void notifyInstalled(Reader condaYaml) {
...
 
                 if (dep instanceof Map) {
+                    //noinspection unchecked
                     if (!((Map<String, Object>) dep).containsKey("pip")) {
-                        continue;
+                        return;
                     }
 
+                    //noinspection unchecked
                     if (!(((Map<String, Object>) dep).get("pip") instanceof List)) {
```
Comment (imakunin) [inline lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java:158]: double lookup

File: lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java [hunk 3]
```diff
@@ -144,21 +149,24 @@ public static void notifyInstalled(Reader condaYaml) {
...
-                        continue;
+                        return;
                     }
 
+                    //noinspection unchecked
                     var pipDeps = (List<Object>) ((Map<String, Object>) dep).get("pip");
```
Comment (imakunin) [inline lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java:163]: triple lookup :)

Comment (mrMakaronka) [reply lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java]: fixed

File: lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java (hunk 2 above)
Comment (mrMakaronka) [reply lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java]: fixed

File: lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java (hunk 1 above)
Comment (mrMakaronka) [reply lzy/worker/src/main/java/ai/lzy/worker/env/CondaPackageRegistry.java]: fixed

**Pull request merged**